frontend/device/input.lua

```diff
@@ -258,20 +258,24 @@ function Input:handleKeyBoardEv(ev)
         end
     end
 
-    if ev.value == EVENT_VALUE_KEY_RELEASE then
-        if keycode == "Light" then
-            return keycode
-        elseif keycode == "Power" then
-            -- Kobo generates Power keycode only, we need to decide whether it's
-            -- power-on or power-off ourselves.
-            if self.device.screen_saver_mode then
-                return "Resume"
-            else
+    if keycode == "Power" then
+        -- Kobo generates Power keycode only, we need to decide whether it's
+        -- power-on or power-off ourselves.
+        if self.device.screen_saver_mode then
+            return "Resume"
+        else
+            if ev.value == EVENT_VALUE_KEY_PRESS then
+                return "PowerPress"
```

Comment:
How about PowerHold? Since we are technically holding the button for 3 seconds ;P